docs/conf.py

```diff
@@ -8,6 +8,8 @@
 linkcheck_ignore = [
     r"http://localhost:\d+/",
     r"https://github.com/mit-nlp/MITIE/releases/download/",
+    r"https://github.com/rasahq/rasa/issues/.*",
+    r"https://github.com/RasaHQ/rasa/issues/.*",
```

Comment:
Have to ignore these bc we can't pass the github auth that lets us send all the requests from the changelog without running into rate limiting

Reply:
wait we already has a linkcheck? why was this not running before lmao?

Reply:
Truly no idea 🙈 clearly someone ran it once!